src/EditorFeatures/CSharpTest/AddParameter/AddParameterTests.cs

```diff
@@ -584,5 +589,1762 @@ void M()
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)]
+        public async Task TestInvocation_Cascading_CrossLanguage()
+        {
+            var code =
+@"
+<Workspace>
+    <Project Language=""Visual Basic"" CommonReferences=""true"" AssemblyName=""VB1"">
+        <Document FilePath=""ReferencedDocument"">
+Namespace N
+    Public Class BaseClass
+        Public Overridable Sub M()
+        End Sub
+    End Class
+End Namespace
+        </Document>
+    </Project>
+    <Project Language=""C#"" CommonReferences=""true""  AssemblyName=""A2"">
+        <ProjectReference>VB1</ProjectReference>
+        <Document>
+namespace N
+{
+    public class Derived: BaseClass
+    {
+        public override void M() { }
+    }
+    public class T
+    {
+        public void Test() { 
+            new Derived().[|M|](1);
+        }
+    }
+}
+        </Document>
+    </Project>
+</Workspace>";
+            var fix =
+@"
+<Workspace>
+    <Project Language=""Visual Basic"" CommonReferences=""true"" AssemblyName=""VB1"">
+        <Document FilePath=""ReferencedDocument"">
+Namespace N
+    Public Class BaseClass
+        Public Overridable Sub M(v As Integer)
+        End Sub
+    End Class
+End Namespace
+        </Document>
+    </Project>
+    <Project Language=""C#"" CommonReferences=""true""  AssemblyName=""A2"">
+        <ProjectReference>VB1</ProjectReference>
+        <Document>
+namespace N
+{
+    public class Derived: BaseClass
+    {
+        public override void M(int v) { }
+    }
+    public class T
+    {
+        public void Test() { 
+            new Derived().M(1);
+        }
+    }
+}
+        </Document>
+    </Project>
+</Workspace>";
+            await TestInRegularAndScriptAsync(code, fix, index: 1);
+        }
+
+        [WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)]
+        public async Task TestInvocation_InvocationStyles_Positional_MoreThanOneArgumentToMuch()
+        {
+            var code =
+@"
+class C
+{
+    void M() { }
+    void Test()
+    {
+        [|M|](1, 2, 3, 4);
+    }
+}";
+            var fix0 =
+@"
+class C
+{
+    void M(int v) { }
+    void Test()
+    {
+        M(1, 2, 3, 4);
+    }
+}";
+            await TestActionCountAsync(code, 1);
+            await TestInRegularAndScriptAsync(code, fix0, index: 0);
+        }
+
+        [WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)]
+        public async Task TestInvocation_InvocationStyles_Positional_WithOptionalParam()
+        {
+            // error CS1501: No overload for method 'M' takes 2 arguments
+            var code =
+@"
+class C
+{
+    void M(int i = 1) { }
+    void Test()
+    {
+        [|M|](1, 2);
+    }
+}";
+            var fix0 =
+@"
+class C
+{
+    void M(int i = 1, int v = 0) { }
```

Comment:
not default?

Reply:
> not default?

I don't understand the question. `void M(int i=1, int v) { }` wouldn't be legal C#.

Reply:
> is it following some kind of code style options

I don't know. I used [SyntaxGenerator.DefaultExpression](https://github.com/dotnet/roslyn/pull/22314/files#diff-cd69c08cefc289bf4f0609d5df526cc5R424).